src/classes/FreshbooksAPI.cls

```diff
@@ -129,7 +129,15 @@ public with sharing class FreshbooksAPI {
 		//client.addChildElement('language',null,null).addTextNode(language);
 		//client.addChildElement('currency_code',null,null).addTextNode(currencyCode);
 		appendTagAndText(client,'notes',a.Freshbooks_Notes__c);
-		appendTagAndText(client,'p_street1',a.BillingStreet);
+		
+                // If we've got two lines of street address, put them into separate lines, if not put into one and clean it up.
+                String[] str = a.BillingStreet.split('\r\n');
```

Comment:
Mac and Linux use only \n so it sounds to me like apex does some cleanup on \r\n during split, since CRLF is only on Windows.